packages/common/src/utils/challenges.ts

```diff
@@ -145,26 +145,3 @@ export const challengeRewardsConfig: Record<
     id: 'trending-underground'
   }
 }
-
-export const sortChallenges = (
```

Comment:
I think we can leave this here and share it? Why did it move?

Also, maybe we can rename this to `makeChallengeSortComparator` or something since it's not actually sorting

Reply:
Names of members to compare changes between using optimistic vs unoptimistic challenges!

Reply:
... wat? I don't know what this means 😅

Reply:
In mobile I want to use the optimistic challenges in the comparator, but in web I want to use the unoptimistic. The object types and members are different (ie. is_disbursed in unoptimistic vs. state === 'disbursed' in optimistic). Figured it was easiest to handle each case individually.

Reply:
ahhhhhhh I see it now

Reply:
yeah this is kinda annoying. I do like keeping this in utils either way even if it means two separate functions - maybe `makeOptimisticChallengeComparator` and `makeUserChallengeComparator`?

Reply:
Fair enough!